### PR TITLE
Update Release Info

### PR DESCRIPTION
**Pull request by FK78**
Resolves 

**Overall change:**
_Update Release Info docs._

**Code changes:**

- _Add in Gahuza STY URLs._
- _Add in UKChina Trad STY URLs._
- _Add in UKChina Simp STY URLs._

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [x] I have assigned this PR to the Simorgh project

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

File: docs/Simorgh-Release-Info.md [hunk 1]
```diff
@@ -51,4 +51,4 @@ Remember to commit and push the changes to both simorghLaunchDates.js and Simorg
...
+| zhongwenTrad | [home](http://localhost:7080/zhongwen/trad)<br/>[articles](http://localhost:7080/zhongwen/articles/c3xd4x9prgyo/trad)<br/>[MAP](http://localhost:7080/zhongwen/trad/uk-23283128)<br/>[PGL](http://localhost:7080/zhongwen/trad/chinese-news-49065935) | [home](https://www.test.bbc.com/zhongwen/trad)<br/>[MAP](https://www.test.bbc.com/zhongwen/trad/uk-23283128)<br/>[PGL](https://www.test.bbc.com/zhongwen/trad/23161412)<br/>[STY](https://www.test.bbc.com/ukchina/simp/uk-qa-23094520) | [home](https://www.stage.bbc.com/zhongwen/trad)<br/>[MAP](https://www.stage.bbc.com/zhongwen/trad/uk-23283128)<br/>[PGL](https://www.stage.bbc.com/zhongwen/trad/23161412) | [home](https://www.bbc.com/zhongwen/trad)<br/>[PGL](https://www.bbc.com/zhongwen/trad/chinese-news-49065935)<br/>[STY](https://www.bbc.com/ukchina/trad/52558470) | __Home__: 31st March 2020<br/>__Articles__: Sept 2019<br/>__PGL__: 24th March 2020<br/>__STY__: 28th May 2020 |
```
Comment (chris-hinds) [inline docs/Simorgh-Release-Info.md]: These updates should be on ukchina not zhongwen

**Review by chris-hinds — CHANGES_REQUESTED** [review]

File: docs/Simorgh-Release-Info.md (hunk 1 above)
Comment (HarveyPeachey) [inline docs/Simorgh-Release-Info.md]: Doesn't this belong in `ukchina` not `zhongwen`?

File: docs/Simorgh-Release-Info.md [hunk 2]
```diff
@@ -44,7 +44,7 @@ Remember to commit and push the changes to both simorghLaunchDates.js and Simorg
...
+| ukchinaTrad | [home](http://localhost:7080/ukchina/trad)<br/>[articles](http://localhost:7080/ukchina/articles/c0e8weny66ko/trad)<br/>[MAP](http://localhost:7080/ukchina/trad/23279012)<br/>[PGL](http://localhost:7080/ukchina/trad/cool-britannia-38434549) | [home](https://www.test.bbc.com/ukchina/trad)<br/>[MAP](https://www.test.bbc.com/ukchina/trad/23279012)<br/>[PGL](https://www.test.bbc.com/ukchina/trad/23099907) | [home](https://www.stage.bbc.com/ukchina/trad)<br/>[MAP](https://www.stage.bbc.com/ukchina/trad/23279012)<br/>[PGL](https://www.stage.bbc.com/ukchina/trad/23099907) | [home](https://www.bbc.com/ukchina/trad)<br/>[PGL](https://www.bbc.com/ukchina/trad/cool-britannia-38434549)<br/>[STY](https://www.bbc.com/ukchina/trad/52558470) | __Home__: 17th March 2020<br/>__Articles__: Sept 2019<br/>__PGL__: 24th March 2020<br/>__STY__: 28th May 2020 |
```
Comment (chris-hinds) [inline docs/Simorgh-Release-Info.md]: can you do the same for ukchinSimp and also add a local test and live link

**Review by chris-hinds — DISMISSED** [review]

**Review by chris-hinds — APPROVED** [review]

**Review by HarveyPeachey — APPROVED** [review]
LGTM

**Pull request merged**